distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -288,99 +289,6 @@ Any.is_a self typ = is_a self typ
 Base_Error.is_a : Any -> Boolean
 Base_Error.is_a self typ = typ==Any || typ==Base_Error
 
-## PRIVATE
-
-   Gets the atom constructor instance for the provided atom.
-
-   Arguments:
-   - atom: The atom to obtain the constructor for.
-get_atom_constructor : Atom -> Any
-get_atom_constructor atom = @Builtin_Method "Meta.get_atom_constructor"
-
-## PRIVATE
-
-   Get the fields for the provided atom.
-
-   Arguments:
-   - atom: The atom to obtain the fields for.
-get_atom_fields : Atom -> Array
-get_atom_fields atom = @Builtin_Method "Meta.get_atom_fields"
```

Comment:
Guys, do you think it is OK to remove methods like this?
- those are usually accessible from types above
- like `Atom.fields`
- so I assume those methods were there only because we had no concept of making things `private`